lib/tasks/delivery.rake

```diff
@@ -1,8 +1,18 @@
 # frozen_string_literal: true
+require 'deliveries_manager'
 
 namespace :delivery do
   desc 'listen to delivery creation commands'
   task subscribe: :environment do
-    DeliveriesManager::CommandReader.new.perform
+
+    conn = Bunny.new(ENV['RABBIT_URL'])
+    conn.start
+    ch = conn.create_channel
+    queue = ch.queue('delivery.status_updated', auto_delete: true)
```

Comment:
why use auto_delete true?

Reply:
You are right, didn't pay much attention to it. This is also the way it is used in message_publisher.rb. I can omit that.
def perform(message)
      conn = Bunny.new(ENV['RABBIT_URL'])
      conn.start
      ch = conn.create_channel
      queue = ch.queue('delivery.status_updated', auto_delete: true)
      exchange = ch.default_exchange
      exchange.publish(message, routing_key: queue.name)
    ensure